Plugin.pm

```diff
@@ -1729,11 +1751,20 @@ sub trackInfoMenu {
 				name => cstring($client, 'PLUGIN_QOBUZ_MANAGE_FAVORITES'),
 				url  => \&QobuzManageFavorites,
 				passthrough => [$args],
-			}
+			} if $args
```

Comment:
`$args` will always be truthy here, as we define `my $args = ()`. It's never undefined. But this should actually be assigned `{}` rather than `()`. And then the test should be `if keys %args`.

Reply:
Done.